indico/modules/rb_new/client/js/common/timeline/MonthlyTimelineContent.jsx

```diff
@@ -75,10 +75,28 @@ export default class MonthlyTimelineContent extends WeeklyTimelineContent {
         const nDays = this.dates.length;
         const daySize = (100 / nDays);
 
+        const {dateRange} = this.props;
+        const emptyDays = dateRange.length === 0 ? [] : this.dates.filter((day) => {
```

Comment:
I think the ternary here makes it much less readable. I'd rather assign an empty list and then, if there are dates, assign the generated value

or is this.dates an empty list in case there's no dateRange? in that case no extra code would be needed